internal/configs/eslint-preset.js

```diff
@@ -42,5 +42,7 @@ module.exports = {
     curly: 'error',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     'no-restricted-imports': ['error', '..'],
+    'react/react-in-jsx-scope': 'off',
+    'react/jsx-uses-react': 'off',
```

Comment:
done in [`e3ec1a5`](https://github.com/iTwin/iTwinUI/pull/2102/commits/e3ec1a595dad7de318bca15d77ed817af4d7a40e)

Reply:
This change required me to also update stories ([`ce12597`](https://github.com/iTwin/iTwinUI/pull/2102/commits/ce1259770a19a55b3f5fc572ca6c6ae915024e76)), since `react-workshop` uses the same eslint preset